### PR TITLE
Delete github_handle when empty field

### PR DESCRIPTION
**Pull request by juuh42dias**
Related #794

### CODE REVIEW

**Comment by carpodaster** [comment]
Hello @juuh42dias! I don't think this PR will solve the underlying problem. See my comment at issue #794.

**Comment by carpodaster** [comment]
Hey @juuh42dias that looks very good now. Do you think you could add a test for that?

File: app/controllers/teams_controller.rb [hunk 1]
```diff
@@ -74,6 +74,7 @@ def set_users
 
     def team_params
       params[:team].fetch(:sources_attributes, {}).delete_if { |key, source| source[:url].empty? }
+      params[:team].fetch(:roles_attributes, {}).delete_if { |key, role| role.present? && role[:github_handle].empty? }
```
Comment (carpodaster) [inline app/controllers/teams_controller.rb]: I'm not really sure what this line and the one above do. That is, I don't know if `#fetch` returns a _copy_ of the nested hash or the actual instance. Depending on what object it returns, deleting keys from it probably doesn't do anything.

In any event, I would remove the line again. The problem (or well: at least _one_ of the problems 😉 ) should be solved by your change in the github_handle concern.

**Comment by carpodaster** [comment]
@juuh42dias are you working on a test? I'm fine with merging without one but if you'd like to go for it that's of course highly appreciated. But we can also add one later.

**Comment by juuh42dias** [comment]
I'll create tests for this feature, is better.

**Comment by klappradla** [comment]
Since this is now lying around for quite a while, what's the state of this @juuh42dias ?

Is the feedback @carpodaster gave implemented / discussed? Are there any open questions or any blockers? Or - also possible of course 😉  - is this no longer relevant?

**Comment by carpodaster** [comment]
@juuh42dias Are you ok if we merge this? I can add the test later as a PR and ping you for a review / walkthrough.

I would remove the last two commits though, the shared example doesn't test anything right now.

**Pull request closed without merging**